src/modules/70-pipeline/utils/runPipelineUtils.ts

```diff
@@ -35,6 +35,13 @@ export function walkObjectRecursively(
   path: string[] = []
 ): void {
   if (typeof obj === 'object' && obj !== null) {
+    const keys = Object.keys(obj)
+
+    // if we have both default key and <+input>.default() then use the later else use default provided as key
+    if (keys.includes('default') && !get(obj, 'value').includes('default')) {
```

Comment:
Instead of const keys here, we can directly use get(obj, 'default')
```suggestion
    if (!isEmpty(get(obj, 'default')) && !get(obj, 'value').includes('default')) {
```